packages/charting/src/components/LineChart/LineChart.styles.ts

```diff
@@ -1,27 +1,27 @@
 import { ILineChartStyleProps, ILineChartStyles } from './LineChart.types';
 
 export const getStyles = (props: ILineChartStyleProps): ILineChartStyles => {
-  const { className, theme, width, height } = props;
+  const { className, theme, width, height, color } = props;
+  const { palette } = theme!;
 
-  const chartWidth = width + 30;
-  const chartPadding = 20;
-  const chartHeight = height + 10;
-  const xOffset = 30;
-  const yOffset = 20;
+  const chartPadding = 30;
+  const scalingVal = 0.3;
+  const xOffset = 20;
 
   return {
     root: [
       'ms-LineChart',
       className,
```

Comment:
Not part of your changes, but className should be the last element in the array.